packages/terra-horizontal-divider/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+ChangeLog
+=========
+
+1.0.0 - (<!--- Insert date using June 28, 2017 format. --->)
```

Comment:
Change this to Unreleased

Reply:
Updated  https://github.com/cerner/terra-core/pull/962/commits/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-c63c93703c3c109e67a8580db8816cfbL1